examples/chip-tool/commands/discover/DiscoverOperationalsCommand.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "DiscoverOperationalsCommand.h"
+#include <commands/common/DeviceScanner.h>
+#include <commands/common/RemoteDataModelLogger.h>
+#include <lib/support/BytesToHex.h>
+
+using namespace ::chip;
+
+CHIP_ERROR DiscoverOperationalsStartCommand::RunCommand()
```

Comment:
So I am curious: what happens if the user does "start-operationals" followed by "start"?

Reply:
I have updated the code such that 'CHIP_ERROR_INCORRECT_STATE' is returned. The user will have to issue the `stop` command first.